omod/src/main/webapp/admin/users/userForm.jsp

```diff
@@ -90,6 +90,10 @@
 			<spring:nestedPath path="user.person.names[0]">
 				<openmrs:portlet url="nameLayout" id="namePortlet" size="full" parameters="layoutMode=edit|layoutShowTable=false|layoutShowExtended=false" />
 			</spring:nestedPath>
+			<tr>
```

Comment:
Can you avoid displaying this for platform versions which do not have the email field?

Reply:
Here is an example: https://github.com/openmrs/openmrs-module-xforms/blob/4.3.14/omod/src/main/java/org/openmrs/module/xforms/page/controller/PatientRegPageController.java#L102

https://github.com/openmrs/openmrs-module-xforms/blob/4.3.14/omod/src/main/webapp/patientReg.jsp#L123

Reply:
would this work 
```
try {
	Field emailField = getEmailField();
	model.addAttribute("email", emailField.get(user));
	model.addAttribute("hasEmailField", true);
} catch (IllegalArgumentException | IllegalAccessException | NullPointerException e) {
	log.warn("Email field not available for setting", e);
	model.addAttribute("hasEmailField", false);
}

```